### DISCUSSION TITLE
[BUG] No chia in a month - 1PB of plots

### DISCUSSION THREAD

**Discussion by berlef80** [Chia-Network/chia-blockchain#8203] [Support]
everything looks like it is farming.  I have gotten chia previously on the same set up other than upgrading to 1.2.3  I have 1PB of plots and it say average win is 7 days, but nothing in 28 days...

**Comment by EVILTIM58** [comment]
I got mine on day 3 with less than a TB in plots.. lol

**Comment by loppefaaret** [comment]
x4 swings is not abnormal.. there is a 2%~ chance to go a month without winning with 1PiB, and the current netspace.
But please keep in mind, that just cause you haven't won, does not make you _more_ likely to win, each round has the same chance of winning.

**Comment by Jacek-ghub** [comment]
@loppefaaret At which point would you consider it being abnormal? I mean, you took that percentage from binomial distribution (you know the drill), so where do you draw the line?  Will you just keep adding more digits after the decimal point, and still call it "not abnormal?"  In about 5 days his percentage will be just 1%, etc.

Just to be on the same page, below is a chart with percentages for winning zero blocks during the second month (for 1PB farm).  Those percentages drop off fast, but as we know, the math will never go to zero.

Another thing is that there are more people with over 1PB farms that voiced exactly the same concerns, and no one addressed them.  I would assume that it would be really easy for Chia team to show some stats (e.g., for farms between 1-10PB, and what is the current distribution of winning blocks for them).  Potentially that would make people sleep better.

![winning 0 blocks](https://user-images.githubusercontent.com/85920330/131078125-a0e858fe-dc12-4957-86e6-27d36b70557a.jpg)

**Reply by loppefaaret** [reply]
personally, ive been down to 0.53% at one point my self - i was biting my nails, spending hours trying to make sure that everything was working as it should. i dont know what would be the line for "abnormal" - but x4 or x5 is not, in my opinion, ive been out there a couple of times already myself.

**Comment by Jacek-ghub** [comment]
@loppefaaret That is good info (you dropping to 0.53%.  Looking at that chart, he is only about 10 days away from the same.  After you reached that 0.53%, how quickly did you get back to about normal on the binomial distribution (I assume you did)?

Also, being there twice would imply that your farm has to be over 1PB, as the official Chia is about 4 months.  It is rather hard to imagine going so low so quickly.  The assumption is that you are right now in the "normal" zone, so those would be rather big swings (both dry and rainy times).

Again, a nice chart that would show that say about 0.01% of farmers hit that level at some point would be nice to see.  People would understand that 1PB farm is actually not that big in 1EB net-space.

To me that also reinforces the notion that Chia UI needs more, better stats to show the health of the farm.

**Comment by loppefaaret** [comment]
it was early i went behind - im not a peta farmer... it took me about 1½-2 months to get back on track, i had a very lucky streak in june... but i havent won since

**Comment by Jacek-ghub** [comment]
With 1PB farm it takes right now 1.5 months to get that low, and as the total net-space didn't increase that much in the last 2 months, it seems to be that there is no time to hit that low point twice, plus have recovery time over another 2 months.  Although, the Apr/May growth was really fast.

**Comment by loppefaaret** [comment]
exactly, my experience was from launch, and into june- now im no longer tracking, my farm is so small it is months just to ettw. and i can only agree that the scenario i base on, is not real now

**Comment by BrandtH22** [comment]
@berlef80 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
